sympy/solvers/ode/single.py

```diff
@@ -425,42 +432,43 @@ def _get_general_solution(self, *, simplify: bool = True):
         return [gensol]
 
 
-
 class AlmostLinear(SinglePatternODESolver):
     r"""
     Solves an almost-linear differential equation.
 
     The general form of an almost linear differential equation is
 
-    .. math:: f(x) g(y) y + k(x) l(y) + m(x) = 0
-                \text{where} l'(y) = g(y)\text{.}
+    .. math:: a(x) g'(f(x)) f'(x) + b(x) g(f(x)) + c(x)
 
-    This can be solved by substituting `l(y) = u(y)`.  Making the given
-    substitution reduces it to a linear differential equation of the form `u'
-    + P(x) u + Q(x) = 0`.
+    This can be solved by substituting `g(f(x)) = u`.  Making the given
+    substitution reduces it to a linear differential equation of the form
+    `a(x) u' + b(x) u + c(x) = 0` which can now be solved for `f(x)`.
```

Comment:
I think it should be:
```
The general form ... is

.. math: ...

where `f(x)` is the function to be solved for (the dependent variable). The substitution `g(f(x)) = u(x)` leads to a linear differential equation for `u(x)` of the form .... This can be solved for `u(x)` by the `first_linear` hint and then `f(x)` is found by solving `g(f(x)) = u(x)`.
```